pkg/actions/extract_index.go

```diff
@@ -20,10 +20,10 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
-	"capabilities-tool/pkg"
+	"opcap/pkg"
 )
 
-const catalogIndex = "audit-catalog-index"
+const catalogIndex = "audit-catalog-one"
```

Comment:
index here should be fine

Reply:
Will be removing this action since it is not being used opcap anymore.